src/main/java/subway/line/LineService.java

```diff
@@ -0,0 +1,50 @@
+package subway.line;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import subway.station.Station;
+import subway.station.StationRepository;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@Transactional(readOnly = true)
+public class LineService {
+    private LineRepository lineRepository;
+    private StationRepository stationRepository;
+
+    public LineService(LineRepository lineRepository, StationRepository stationRepository) {
+        this.lineRepository = lineRepository;
+        this.stationRepository = stationRepository;
+    }
+
+    @Transactional
+    public LineResponse saveLine(LineRequest lineRequest) {
+        Line line = lineRepository.save(new Line(lineRequest));
+        return createLineResponse(line);
+    }
+
+    public List<LineResponse> findAllLines() {
+        return lineRepository.findAll().stream()
+                .map(this::createLineResponse)
+                .collect(Collectors.toList());
+    }
+
+    @Transactional
+    public void deleteLineById(Long id) {
+        lineRepository.deleteById(id);
+    }
+
+    private LineResponse createLineResponse(Line line) {
+        Station up = stationRepository.getOne(line.getUpStationId());
```

Comment:
```suggestion
Station up = stationRepository.findById(line.getUpStationId()).orElseThrow(() -> new IllegalArgumentException("역이 존재하지 않음 : " + line.getUpStationId()));
```

위와 같은 형태로 사용하시면 됩니다 😄  [참고](https://sundries-in-myidea.tistory.com/91)